yt_dlp/extractor/instagram.py

```diff
@@ -216,13 +219,12 @@ class InstagramIOSIE(InfoExtractor):
 
     def _get_id(self, id):
         """Source: https://stackoverflow.com/questions/24437823/getting-instagram-post-url-from-media-id"""
-        chrs = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_'
         media_id = int(id.split('_')[0])
         shortened_id = ''
         while media_id > 0:
             r = media_id % 64
             media_id = (media_id - r) // 64
-            shortened_id = chrs[r] + shortened_id
+            shortened_id = _ENCODING_CHARS[r] + shortened_id
         return shortened_id
```

Comment:
not necessary, but it may make sense to create `utils.decode_base_n` with this

Reply:
I don't think its required since `utils.decode_base_n` seems like a base module and a particular small usability func like `_get_pk` can exist independentely

Reply:
cannot resolve